appserver/resources/resources-connector/src/test/resources/DomainTest.xml

```diff
@@ -153,6 +153,7 @@
         <jvm-options>--add-opens=java.base/java.util=ALL-UNNAMED</jvm-options>
         <jvm-options>--add-opens=java.rmi/sun.rmi.transport=ALL-UNNAMED</jvm-options>
         <jvm-options>--add-opens=java.naming/javax.naming.spi=ALL-UNNAMED</jvm-options>
+        <jvm-options>--add-exports java.naming/com.sun.jndi.ldap=ALL-UNNAMED</jvm-options>
```

Comment:
Does add-exports really use a space instead of equal sign as add-opens does?

Reply:
The examples I looked at all used a space indeed, and I tested it works. But perhaps the equal sign works equally well. I'll test it.